extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -668,62 +805,82 @@ public function searchSupportsCurrentLanguage() {
 
 	/**
 	 * Determines if a given language code is supported for dynamic search
+	 *
 	 * @param string $languageCode
+	 *
 	 * @return boolean
 	 */
 	public function searchSupportsLanguageCode( $languageCode ) {
 		return in_array( $languageCode, $this->getGlobal( 'WikiaSearchSupportedLanguages' ) );
 	}
 
 	public function getThumbnailUrl( $pageId, $dimensions = null ) {
-		$width = (isset( $dimensions[ 'width' ] ) ) ? $dimensions[ 'width' ] : static::THUMB_DEFAULT_WIDTH;
-		$height = (isset( $dimensions[ 'height' ] ) ) ? $dimensions[ 'height' ] : static::THUMB_DEFAULT_HEIGHT;
+		$width = ( isset( $dimensions['width'] ) ) ? $dimensions['width'] : static::THUMB_DEFAULT_WIDTH;
+		$height = ( isset( $dimensions['height'] ) ) ? $dimensions['height'] : static::THUMB_DEFAULT_HEIGHT;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "height" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)